internal/stolonctl/pkg/cluster/status.go

```diff
@@ -106,25 +107,18 @@ type Status struct {
 	ClusterData *cluster.ClusterData
 }
 
-func (s *Status) getMasterStatus() (*MasterStatus, error) {
+func (s *Status) getMasterStatus() (*crd.MasterStatus, error) {
 	for _, pod := range s.PodsStatus {
-		if pod.PodIP == s.ClusterData.KeepersState[s.ClusterData.ClusterView.Master].ListenAddress {
-			return &MasterStatus{
-				PodName: pod.Name,
-				Healthy: s.ClusterData.KeepersState[s.ClusterData.ClusterView.Master].Healthy,
-				HostIP:  pod.HostIP,
-			}, nil
+		if _, ok := s.ClusterData.KeepersState[s.ClusterData.ClusterView.Master]; ok {
+			if pod.PodIP == s.ClusterData.KeepersState[s.ClusterData.ClusterView.Master].ListenAddress {
```

Comment:
you could avoid double access by storing the result in a variable:
```go
if master, ok := s.ClusterData.KeepersState[s.ClusterData.ClusterView.Master]; ok {
			if pod.PodIP == master.ListenAddress {
```
I'd also invert the condition to check for value: `master != nil`